great_ape_safe/ape_api/uni_v2.py

```diff
@@ -155,7 +156,8 @@ def swap_exact_eth_for_tokens(self, amountIn, path, destination=None):
         address to,
         uint256 deadline
         """
-        self.router.swapExactETHForTokens(
+        signature = getattr(self.router, f'swapExact{self.native_symbol}ForTokens')
```

Comment:
this is a nice solution. at least they actually changed the func names, i remember seeing pancakeswap just using the func names with ETH even though it really is BNB...

Reply:
ya some routers use the right native symbol and some dont, this is the best solution I could think of to address this inconsistency without duping a lot of code